Content/Items/Currency/LimpidShard.cs

```diff
@@ -0,0 +1,22 @@
+﻿using PathOfTerraria.Common.Enums;
+using PathOfTerraria.Core.Items;
+
+namespace PathOfTerraria.Content.Items.Currency;
+
+internal class LimpidShard : CurrencyShard
+{
+	public override bool CanRightClick()
+	{
+		PoTInstanceItemData data = Main.LocalPlayer.HeldItem.GetInstanceData();
+		return base.CanRightClick() && data.Rarity is ItemRarity.Magic or > ItemRarity.Rare;
```

Comment:
Isn't `data.Rarity is ItemRarity.Magic or > ItemRarity.Rare` equivalent to `data.Rarity is ItemRarity.Magic or ItemRarity.Unique`? 
And, according to the description, that's not the intent - do `is Magic or Rare` instead. > is just confusing.

Reply:
Oh, weird. Not sure where `>` came from.